.github/workflows/release.yml

```diff
@@ -9,20 +9,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
       - name: Setup Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
```

Comment:
Figured I may as well bump these while I was here for consistency with the `ci.yml` flow.